<details>

```
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-2:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-4:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-6:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-10:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-22:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-23:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-24:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-20:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-16:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-14:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-5:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-7:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-21:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-18:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-11:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-12:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-19:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
STDERR: Exception in thread TestRunnerManager-8:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
Traceback (most recent call last):
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent

STDERR: Exception in thread TestRunnerManager-17:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 386, in run
    new_state = self.wait_event()
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 461, in wait_event
    return f(*data)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 592, in test_ended
    stack=result.stack)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 51, in inner
    data = converter.convert(*args, **kwargs)
  File "/home/servo/buildbot/slave/linux-rel-wpt/build/python/_virtualenv/lib/python2.7/site-packages/mozlog/logtypes.py", line 96, in convert
    raise TypeError("Unrecognised argument %s" % key)
TypeError: Unrecognised argument known_intermittent
```

</details>

--- a/python/requirements.txt
+++ b/python/requirements.txt
@@ -32,4 +32,7 @@ boto3 == 1.4.4
 # https://github.com/servo/servo/pull/18942
 certifi
 
+# For Python3 compatibility
+six
+
 -e python/tidy

--- a/python/servo/bootstrap_commands.py
+++ b/python/servo/bootstrap_commands.py
@@ -244,7 +244,7 @@ class MachCommands(CommandBase):
 
             with open(path.join(preload_path, preload_filename), 'w') as fd:
                 json.dump(entries, fd, indent=4)
-        except ValueError, e:
+        except ValueError as e:
             print("Unable to parse chromium HSTS preload list, has the format changed?")
             sys.exit(1)
 

--- a/python/servo/command_base.py
+++ b/python/servo/command_base.py
@@ -7,6 +7,8 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+from __future__ import print_function
+
 from errno import ENOENT as NO_SUCH_FILE_OR_DIRECTORY
 from glob import glob
 import shutil
@@ -350,15 +352,15 @@ class CommandBase(object):
                 version_line = subprocess.check_output(["rustup" + BIN_SUFFIX, "--version"])
             except OSError as e:
                 if e.errno == NO_SUCH_FILE_OR_DIRECTORY:
-                    print "It looks like rustup is not installed. See instructions at " \
-                          "https://github.com/servo/servo/#setting-up-your-environment"
-                    print
+                    print("It looks like rustup is not installed. See instructions at "
+                          "https://github.com/servo/servo/#setting-up-your-environment")
+                    print()
                     return 1
                 raise
             version = tuple(map(int, re.match("rustup (\d+)\.(\d+)\.(\d+)", version_line).groups()))
             if version < (1, 11, 0):
-                print "rustup is at version %s.%s.%s, Servo requires 1.11.0 or more recent." % version
-                print "Try running 'rustup self update'."
+                print("rustup is at version %s.%s.%s, Servo requires 1.11.0 or more recent." % version)
+                print("Try running 'rustup self update'.")
                 return 1
             toolchain = self.toolchain()
             if platform.system() == "Windows":

--- a/python/servo/lints/wpt_lint.py
+++ b/python/servo/lints/wpt_lint.py
@@ -7,6 +7,8 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+from __future__ import print_function
+
 import os
 import sys
 
@@ -20,7 +22,7 @@ class Lint(LintRunner):
     def _get_wpt_files(self, suite):
         working_dir = os.path.join(WPT_PATH, suite, '')
         file_iter = self.get_files(working_dir, exclude_dirs=[])
-        print '\nRunning the WPT lint on %s...' % working_dir
+        print('\nRunning the WPT lint on %s...' % working_dir)
         for f in file_iter:
             if filter_file(f):
                 yield f[len(working_dir):]

--- a/python/servo/mutation/mutator.py
+++ b/python/servo/mutation/mutator.py
@@ -7,6 +7,8 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+from __future__ import print_function
+
 import fileinput
 import re
 import random
@@ -28,7 +30,7 @@ def init_variables(if_blocks):
 def deleteStatements(file_name, line_numbers):
     for line in fileinput.input(file_name, inplace=True):
         if fileinput.lineno() not in line_numbers:
-            print line.rstrip()
+            print(line.rstrip())
 
 
 class Strategy:
@@ -48,7 +50,7 @@ class Strategy:
             for line in fileinput.input(file_name, inplace=True):
                 if fileinput.lineno() == mutation_line_number:
                     line = re.sub(self._replace_strategy['regex'], self._replace_strategy['replaceString'], line)
-                print line.rstrip()
+                print(line.rstrip())
             return mutation_line_number
 
 

--- a/python/servo/util.py
+++ b/python/servo/util.py
@@ -136,16 +136,16 @@ def download(desc, src, writer, start_byte=0):
 
         if not dumb:
             print()
-    except urllib.error.HTTPError, e:
+    except urllib.error.HTTPError as e:
         print("Download failed ({}): {} - {}".format(e.code, e.reason, src))
         if e.code == 403:
             print("No Rust compiler binary available for this platform. "
                   "Please see https://github.com/servo/servo/#prerequisites")
         sys.exit(1)
-    except urllib.error.URLError, e:
+    except urllib.error.URLError as e:
         print("Error downloading {}: {}. The failing URL was: {}".format(desc, e.reason, src))
         sys.exit(1)
-    except socket_error, e:
+    except socket_error as e:
         print("Looks like there's a connectivity issue, check your Internet connection. {}".format(e))
         sys.exit(1)
     except KeyboardInterrupt:

--- a/python/tidy/servo_tidy/tidy.py
+++ b/python/tidy/servo_tidy/tidy.py
@@ -7,6 +7,8 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+from __future__ import print_function
+
 import contextlib
 import fnmatch
 import imp
@@ -514,7 +516,7 @@ def check_manifest_dirs(config_file, print_text=True):
         lines = conf_file.splitlines(True)
 
     if print_text:
-        print '\rChecking the wpt manifest file...'
+        print('\rChecking the wpt manifest file...')
 
     p = parser.parse(lines)
     paths = rec_parse(wpt_path("web-platform-tests"), p)
@@ -908,7 +910,7 @@ def check_config_file(config_file, print_text=True):
         lines = conf_file.splitlines(True)
 
     if print_text:
-        print '\rChecking the config file...'
+        print('\rChecking the config file...')
 
     config_content = toml.loads(conf_file)
     exclude = config_content.get("ignore", {})
@@ -995,7 +997,7 @@ def parse_config(config_file):
 
 def check_directory_files(directories, print_text=True):
     if print_text:
-        print '\rChecking directories for correct file extensions...'
+        print('\rChecking directories for correct file extensions...')
     for directory, file_extensions in directories.items():
         files = sorted(os.listdir(directory))
         for filename in files:
@@ -1015,7 +1017,7 @@ def collect_errors_for_files(files_to_check, checking_functions, line_checking_f
     if not has_element:
         raise StopIteration
     if print_text:
-        print '\rChecking files for tidiness...'
+        print('\rChecking files for tidiness...')
 
     for filename in files_to_check:
         if not os.path.exists(filename):
@@ -1037,7 +1039,7 @@ def collect_errors_for_files(files_to_check, checking_functions, line_checking_f
 
 def get_dep_toml_files(only_changed_files=False):
     if not only_changed_files:
-        print '\nRunning the dependency licensing lint...'
+        print('\nRunning the dependency licensing lint...')
         for root, directories, filenames in os.walk(".cargo"):
             for filename in filenames:
                 if filename == "Cargo.toml":
@@ -1136,11 +1138,11 @@ def scan(only_changed_files=False, progress=True, stylo=False):
     error = None
     for error in errors:
         colorama.init()
-        print "\r\033[94m{}\033[0m:\033[93m{}\033[0m: \033[91m{}\033[0m".format(*error)
+        print("\r\033[94m{}\033[0m:\033[93m{}\033[0m: \033[91m{}\033[0m".format(*error))
 
-    print
+    print()
     if error is None:
         colorama.init()
-        print "\033[92mtidy reported no errors.\033[0m"
+        print("\033[92mtidy reported no errors.\033[0m")
 
     return int(error is not None)

